### DISCUSSION TITLE
Interface to Medtronic Guardian 4 Transmitter

### DISCUSSION THREAD

**Discussion by phicks1** [Navid200/xDrip#174] [Ideas]
I have recently started using the new Medtronic Guardian 4 sensors & transmitter with my 670G pump. It would be great if the transmitter could be integrated directly with Xdrip.
There is a section in the transmitter manual “Open Source Software Disclosure”
This states:
“This document identifies the OSS that may be separately called, executed etc by this product”
The source / object code …can be obtained here: http://www.ouah.org/ogay/hmac/

This leads to https://github.com/ogay/hmac

Does this help develop an interface between Xdrip & the Guardian 4 transmitter?

**Comment by Navid200** [comment]
I think we already have an open issue for this:
https://github.com/NightscoutFoundation/xDrip/issues/131
